pennylane/qinfo/transforms.py

```diff
@@ -396,3 +396,163 @@ def wrapper(*args, **kwargs):
         return _compute_cfim(p, j)
 
     return wrapper
+
+
+def fidelity(qnode0, qnode1, wires0, wires1):
```

Comment:
Question: what's the behaviour if the user passes in the same QNode with different QNode parameters? Does it work as expected?

Reply:
I admit I didn't try but it should be fine, but you have to pass it twice. We could imagine a version where only one qnode is needed